v21.1/import-into.md

```diff
@@ -164,6 +163,8 @@ Pausing and then resuming an `IMPORT INTO` job will cause it to restart from the
 
 The following provide connection examples to cloud storage providers. For more information on connecting to different storage options, read [Use Cloud Storage for Bulk Operations](use-cloud-storage-for-bulk-operations.html).
 
+{% include {{ page.version.version }}/import-into-offline.md %}
```

Comment:
from style POV this is probably the one I like the least tbh but if we do have to keep it is there a way to make every instance on this page use the include?  tons o repetition